apps/settings/js/carrier.js

```diff
@@ -256,7 +257,7 @@ var CarrierSettings = (function() {
     function onSubmit() {
       setWarningDialogState(false);
       setState(true);
-      explanationItem.hidden = false;
+      explanationItem.removeAttribute('hidden');
```

Comment:
Switching to a `details` node, so I think it's necessary to change to remove/setAttribute.